code/modules/antagonists/cult/blood_magic.dm

```diff
@@ -644,6 +644,10 @@
 	desc = "A spell that will absorb blood from anything you touch.<br>Touching cultists and constructs can heal them.<br><b>Clicking the hand will potentially let you focus the spell into something stronger.</b>"
 	color = "#7D1717"
 
+/obj/item/melee/blood_magic/manipulator/examine(mob/user)
+	. = ..()
+	to_chat(user, "<span class='cultitalic'>The [name] currently has <b>[uses]</b> blood charges left.</span>")
```

Comment:
this should have a cultist check

Reply:
I'm not sure if there's any way for a non-cultist to even examine the aura but I'll put it in for good measure I guess